src/sentry/migrations/0439_backfill_notification_user.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 2.2.28 on 2023-05-09 19:51
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+ACTOR_TEAM_TYPE = 0
+ACTOR_USER_TYPE = 1
+
+
+def backfill_notification_user(apps, schema_editor):
+    NotificationSetting = apps.get_model("sentry", "NotificationSetting")
+    Actor = apps.get_model("sentry", "Actor")
+    for ns in RangeQuerySetWrapperWithProgressBar(
+        NotificationSetting.objects.filter(user_id=None, team_id=None)
+    ):
```

Comment:
3,939 records affected

Reply:
Tbqh I'd skip using the `RangeQuerySetWrapperWithProgressBar` in that case